components/event-footer/EventFooter.vue

```diff
@@ -44,7 +50,15 @@
     padding-bottom: 1.4rem;
   }
 
+  @media(min-width: 30rem) {
```

Comment:
Guess we can use `--sm-viewport` here now.

Reply:
👍